tests/foreman/api/test_role.py

```diff
@@ -622,12 +858,78 @@ def test_positive_user_group_users_access_as_org_admin(self):
             3. Create two users without assigning roles while creating them
             4. Assign Organization A and Location A to both users
             5. Create an user group with above two users
+            6. Assign Org Admin role to User Group
 
         :expectedresults: Both the user should have access to the resources of
             organization A and Location A
 
         :CaseLevel: System
         """
+        org_admin = self.create_org_admin(
+            orgs=[self.org1.id], locs=[self.loc1.id])
+        user1_login = gen_string('alpha')
+        user1_pass = gen_string('alphanumeric')
+        user1 = entities.User(
+            login=user1_login,
+            password=user1_pass,
+            auth_source=1,
+            organization=[self.org1.id],
+            location=[self.loc1.id]
+        ).create()
+        self.assertEqual(user1_login, user1.login)
+        user2_login = gen_string('alpha')
+        user2_pass = gen_string('alphanumeric')
+        user2 = entities.User(
+            login=user2_login,
+            password=user2_pass,
+            auth_source=1,
+            organization=[self.org1.id],
+            location=[self.loc1.id]
+        ).create()
+        self.assertEqual(user2_login, user2.login)
+        ug_name = gen_string('alpha')
+        ug1 = entities.UserGroup(
+            name=ug_name,
+            role=[org_admin['id']],
+            user=[user1.id, user2.id]
+        ).create()
+        self.assertEqual(ug1.name, ug_name)
+        sc1 = ServerConfig(
+            auth=(user1_login, user1_pass),
+            url=ServerConfig.get().url,
+            verify=False
+        )
+        with self.assertNotRaises(HTTPError):
+            entities.Domain(sc1).search(
+                query={
+                    'organization-id': self.org1.id,
+                    'location-id': self.loc1.id
+                }
+            )
```

Comment:
Checking no errors were risen is not enough - please also ensure there're some results returned.

Reply:
Actually, Its the test of user be able to access Domains, no matter if there is present anything. IF user doesnt have access to that org and location, it will return access error as HTTP error.

Reply:
Ok, lemme rephrase - what if no errors are returned but the resulting list is always empty? Pretty often happens to CLI tests - if user doesn't have enough permissions some `list` command may return him empty list instead of failing. That's what i'm trying to avoid and cover.

Reply:
Ok, I will create some entities and see if those are returned.